src/test/java/uk/nhs/prm/e2etests/test/RepositoryE2ETest.java

```diff
@@ -712,4 +708,36 @@ void shouldSendUnexpectedMessageFormatsThroughToEhrTransferServiceDeadLetterQueu
             assertThat(ehrTransferServiceParsingDeadLetterQueue.getMessageContaining(message)).isNotNull();
         });
     }
+
+    @Test
+    @Timeout(value = 60, unit = SECONDS)
+    void shouldTransferOut20EHRsWithin1Minute() {
+        String nhsNumberForTestPatient = "9727018076";
+        String previousGpForTestPatient = "M85019";
+        String asidCodeForTestPatient = "200000000149";
+        List<String> outboundConversationIds = Stream.generate(() -> (UUID.randomUUID().toString())).limit(20).toList();
+
+        outboundConversationIds.forEach(conversationId -> {
+            EhrRequestTemplateContext ehrRequestTemplateContext = EhrRequestTemplateContext
+                    .builder()
+                    .outboundConversationId(conversationId.toUpperCase())
+                    .nhsNumber(nhsNumberForTestPatient)
+                    .newGpOdsCode(previousGpForTestPatient)
+                    .asidCode(asidCodeForTestPatient)
+                    .build();
+
+            String ehrRequestMessage = this.templatingService.getTemplatedString(EHR_REQUEST, ehrRequestTemplateContext);
+
+            mhsInboundQueue.sendMessage(ehrRequestMessage, conversationId);
```

Comment:
Could we add `this` please, so that when this class grows in size we're able to directly spot it's reference a member!